content/faq/questions/dashboard-faq.md

```diff
@@ -22,7 +22,7 @@ You can read more [here](/guides/dashboard/introduction).
 
 After recording your tests, you will see them in the
 [Dashboard](https://on.cypress.io/dashboard) and in the "Runs" tab of the
-Cypress App.
+Cypress Test Runner.
```

Comment:
Same idea here, since the "Runs" tab is visible in the entire app, I don't think we need to call out Test Runner. We can just leave it as "...and in the "Runs" tab of Cypress."